src/tools/bin/plotNumericalHeating

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+#
+# Copyright 2015 Richard Pausch
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+
+__doc__ = '''
+This program compares the energy evolution of two PIConGPU runs.
+Just give the directory of two PIConGPU runs and this program will
+get all values needed if they are available.
+You need to activate `--energy_fields`, `--energy_e` and `--energy_i`
```

Comment:
It contains currently both electrons and ions. 
It would be better, if it would cover all (0,1,2,-N) species.